src/index.css

```
@@ -507,8 +509,9 @@ html:has(body[data-scroll-locked]) {
   --overlay: 0 0% 0%;
   --overlay-foreground: 0 0% 100%;
 
-  /* Typography stacks (theme presets may swap families). */
-  --font-sans: ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', sans-serif;
+  /* Typography stacks (theme presets may swap families). Inter leads the
+     latin stack for the premium SaaS voice; CJK falls through to the OS face. */
+  --font-sans: 'Inter Variable', ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', 'PingFang SC', 'Hiragino Sans GB', 'Microsoft YaHei', sans-serif;
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)